core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.spec.ts

```diff
@@ -156,13 +158,15 @@ quot;img_20200630_114637_c2ek92uvb8_height_326_width_490.png&amp;quot;\
     service.toggleCopyMode();
     expect(service.copyModeActive).toBe(true);
 
+    /* eslint-disable oppia/no-multiline-disable */
     /* eslint-disable quotes */
```

Comment:
Why do we even need this -- can we get rid of it? Ideally, try to remove pragmas where possible.